extensions/data-plane/data-plane-kafka/src/main/java/org/eclipse/edc/dataplane/kafka/DataPlaneKafkaExtension.java

```diff
@@ -31,6 +32,11 @@ public class DataPlaneKafkaExtension implements ServiceExtension {
 
     public static final String NAME = "Data Plane Kafka";
 
+    private static final int DEFAULT_PART_SIZE = 5;
+
+    @Setting
```

Comment:
please add the `value`, `type`, `defaultValue` and `min` attributes to the annotation

Reply:
Also copied from `DataPlaneHttpExtension`.